index.html

```diff
@@ -320,11 +320,77 @@ <h2 class="food-spin__location-title js-food-spin-location-title">Pick Up </h2>
         </svg>
       </button>
     </section>
+      <div class="cart-summary js-cart-summary" data-open="true">
```

Comment:
Si usas `open` de `dialog` en el js para este div, ¿por que no es un `dialog`? Si no vas a usar dialog, ¿por que el `open`?
```suggestion
      <dialog class="cart-summary js-cart-summary" data-open="true">
```